app/models/questionnaire.rb

```diff
@@ -16,7 +16,7 @@ class Questionnaire < ApplicationRecord
 
   validates :name, presence: true, uniqueness: true
   validates :content, presence: true
-  validates :key, presence: true, uniqueness: true, format: { with: /\A[a-z]+[a-z_0-9]*\Z/ }
```

Comment:
Correct. Because some of them were named `01_demographics` and stuff. 

The reason for not allowing a key to start with a number is that converting a key to a symbol is then no longer possible. Symbols in Ruby can't start with a digit because of syntax reasons.
```ruby
[1] pry(main)> :01_demographics
(eval):2: trailing `_' in number
:01_demographics
   ^
SyntaxError: unexpected integer literal, expecting literal content or terminator or tSTRING_DBEG or tSTRING_DVAR
```

However, I don't believe that we symbolize the key of a questionnaire anywhere. Feel free to double check this.